pkg/ignition/ignition.go

```diff
@@ -27,11 +27,15 @@ const (
 	// RenderedWorkerPrefix allows identification of the rendered worker MachineConfig, the combination of all worker
 	// MachineConfigs.
 	RenderedWorkerPrefix = "rendered-worker-"
+	// KubeletCACertPath is the path to the kubelet's client CA certificate as defined in ignition
+	KubeletCACertPath = "/etc/kubernetes/kubelet-ca.crt"
+	// CloudConfigPath is the path to the cloud config file as defined in ignition
+	CloudConfigPath = "/etc/kubernetes/cloud.conf"
 )
 
 // Ignition is a representation of an Ignition file
 type Ignition struct {
-	config ignCfgTypes.Config
+	Config ignCfgTypes.Config
```

Comment:
Instead of making this public, please add a method such as `GetFiles` or `GetFileList`